server/routes/findInterventions/searchResultsPresenter.ts

```diff
@@ -1,75 +1,12 @@
-import { Eligibility, Intervention } from '../../services/interventionsService'
-import { SummaryListItem } from '../../utils/summaryList'
-import utils from '../../utils/utils'
-
-export interface SearchResultPresenter {
-  title: string
-  href: string
-  body: string
-  summary: SummaryListItem[]
-}
+import { Intervention } from '../../services/interventionsService'
+import InterventionDetailsPresenter, { InterventionDetails } from './InterventionDetailsPresenter'
```

Comment:
The file name (and the test file name) should be `lowerCamelCase`.

Reply:
Oops, don't know how I did this, I blame Fridays.